src/tiled/commandlineparser.cpp

```diff
@@ -100,8 +98,8 @@ bool CommandLineParser::parse(const QStringList &arguments)
         for (int i = 1; i < arg.length(); ++i) {
             const QChar c = arg.at(i);
             if (!handleShortOption(c)) {
-                qWarning().nospace() << "Unknown short argument " << index
-                                     << '.' << i << ": " << c;
+                qWarning()
+                    << qPrintable(tr("Unknown short argument %1.%2: %3").arg(index).arg(i).arg(c));
```

Comment:
bjorn: it 'drawn' by white color. I have no ideas why. It really exists in the file.